src/test/resources/feature/2-mittente/3_downloadFile/014_1_downloadFileDettaglioNotificaMittente.feature

```diff
@@ -8,6 +8,9 @@ Feature: Mittente scarica tutti i file all'interno di una notifica
   Scenario: PN-9327 - Mittente scarica attestazioni
     Given PA - Si effettua la login tramite token exchange, e viene visualizzata la dashboard
     When Creo in background una notifica con un destinatario e un documento tramite API REST
+      | avvisoPagoPa | false |
+      | F24          | false |
+      | entrambi     | true  |
     Then Attendo 5 minuti e verifico in background che la notifica sia stata creata correttamente
```

Comment:
insert here polling for verify create notification

Reply:
In blocked for now